Support/Multitenancy/Ironic_bmo-mult-instances/clean.sh

```diff
@@ -27,7 +27,7 @@ sudo rm -rf  /opt/mypool
 #sudo rm -rf /opt/metal3-dev-env
 sudo rm -rf /opt/metal3-dev-env/ironic/virtualbmc/
 sudo podman stop -a
-sudo podman rmi $(sudo podman images -qa) -f
+sudo podman rmi "$(sudo podman images -qa)" -f
```

Comment:
In this case, quotes are wrong. It makes the command go from: `podman rmi xxx yyy zzz` (ie. removing 3 images) to `podman rmi "xxx yyy zzz"` ie. removing 1 image with malformatted reference format.

Reply:
if I don't do that I am getting this 

In ./Support/Multitenancy/capi_capm3-mult-instances/clean.sh line 30:
sudo podman rmi $(sudo podman images -qa) -f
                ^-----------------------^ SC2046 (warning): Quote this to prevent word splitting.

Reply:
I did what shellcheck told me to do in the first place, as discussed over the slack, I am ignoring shellcheck here.

Reply:
95% of the cases quoting is the correct action, but in this specific case, podman is capable of taking in an arbitrary list of container references, and delete them all. This shellcheck can't know, and it assumes you're trying to pass the subshell output as a single argument.

Two options here:
1. ignore the shellcheck warning here
2. convert the code into for a loop, that processes the output one by one

Ignoring the warning is the better choice here.